.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,63 @@
+name: E2E test on DevNet
+
+# Only trigger, when the build workflow succeeded, and allow manual triggering.
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Test, build and push artifacts"]
+    types:
+      - completed
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+    needs: build-images
+    if: contains(github.event.pull_request.labels.*.name, 'e2e-devnet-test')
```

Comment:
Can you document the use of `e2e-devnet-test` somewhere?

---

Nvm: https://github.com/pokt-network/pocket/pull/761